resources_pages/install_ds_stack_mac.md

```diff
@@ -604,46 +620,48 @@ R -q -e "installed.packages()[,c(Package, Version)]"  # For R packages
 Checking program and package versions...
 
 ## Operating system
-ProductName:    Mac OS X
-ProductVersion: 10.15.5
-BuildVersion:   19F101
+ProductName:	macOS
+ProductVersion:	11.4
+BuildVersion:	20F71
+
+MISSING You need macOS Catalina (10.15.x).
```

Comment:
Note to self to fix this in the script. For the the versions below, I will keep some as missing just to show students what it looks like; i'll change that manually later.

Reply:
Let's scope this PR to not include the script, and make a new PR for the script.